src/libs/OptionsListUtils.ts

```diff
@@ -372,7 +373,7 @@ function getPersonalDetailsForAccountIDs(accountIDs: number[] | undefined, perso
         if (!cleanAccountID) {
             return;
         }
-        let personalDetail: OnyxEntry<PersonalDetails> = personalDetails[accountID];
+        let personalDetail: OnyxEntry<PersonalDetails> = personalDetails[accountID] ?? undefined;
```

Comment:
Unneeded change. This can't be `null`, the nullish fallback is not necessary

Reply:
It can be null, because right now the type of `personalDetails` is

```
type PersonalDetailsList = Record<string, PersonalDetails | null>;
```

I've seen[ this comment](https://github.com/Expensify/App/pull/42772#discussion_r1636784417) and you might be right, but i really don't want to touch this type in this PR, since i didn't create it. This might cause more regressions.

As of now, we need this nullish fallback.